src/pytorch_tabular/utils.py

```diff
@@ -8,7 +8,7 @@
 import pandas as pd
 import torch
 import torch.nn as nn
-from lightning_lite.utilities.cloud_io import get_filesystem
+from lightning_fabric.utilities.cloud_io import get_filesystem
```

Comment:
```suggestion
from pytorch_lightning.utilities.imports import package_available
if package_available("lightning_fabric"):
    from lightning_fabric.utilities.seed import seed_everything
elif package_available("lightning_fabric"):
    from lightning_lite.utilities.seed import seed_everything
```

Reply:
`lightnng_fabric` is something that one will have to install separately?  Then we would have to include it in the requirements as well

Reply:
It is part of PL 1.9 the same way as lite was pert of PL 1.8

Reply:
this shall be added too